lib/packwerk/validator_interface.rb

```diff
@@ -0,0 +1,44 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "constant_resolver"
+require "pathname"
+require "yaml"
+
+module Packwerk
+  module ValidatorInterface
+    extend ActiveSupport::Autoload
+    autoload :Helpers
+
+    extend T::Sig
+    extend T::Helpers
+
+    include Helpers
```

Comment:
This will load helpers immediately, so there is no need to autoload it. Also curious if we should leave it up to the concrete implementation to include this if they need it?

Reply:
When you say there is no need to autoload it -- do you mean replace with a vanilla require statement (since something needs to require it somehow)? I made that change.

Reply:
I'm inclined to keep it because the fact that it's in a separate module is mostly just an implementation detail -- in fact I wonder if it would be better to just inline it into `AbstractValidator`. I think a lot of validators will need these methods (as the existing ones did) and wanted to avoid an extra step for the clients to be able to implement they validation.

That being said I'm open to changing this so the concrete implementation has to include it if you feel strongly about it!